scripts/evaluate_bidaf.sh

```diff
@@ -28,24 +28,27 @@ bidaf_input=${input_file_prefix}_with_paras_${run_name}.jsonl
 if [ ! -f ${input_file_with_hits} ]; then
 	python arc_solvers/processing/add_retrieved_text.py \
 		${input_file} \
-		${input_file_with_hits}
+		${input_file_with_hits}.$$
+	mv ${input_file_with_hits}.$$ ${input_file_with_hits}
```

Comment:
Aah. This is a good trick.

Reply:
This should be done with a makefile. That would have the added benefit that when one of the earlier files changes, it knows to recompute the later ones. But I didn't want to change so much all at the same time.